src/ring_theory/discrete_valuation_ring.lean

```diff
@@ -169,7 +170,7 @@ begin
       exact (hϖ.not_unit (is_unit_of_mul_is_unit_left H0)).elim } }
 end
 
-/-- Implementation detail: an integral domain in which there is a unit `p`
+/-- Implementation detail: an integral domain in which there is an irreducible element `p`
```

Comment:
```suggestion
/-- An integral domain in which there is an irreducible element `p`
```

Reply:
Also, should we rename this from `ufd` to `unique_factorization_monoid`? (I'm assuming it's namespaced.)

Reply:
Well this is a bit more complicated than it sounds. We are currently in the namespace `discrete_valuation_ring.has_unit_mul_pow_irreducible_factorization` and the type of `ufd` is `∀ {R : Type u_2} [_inst_1 : integral_domain R],
    has_unit_mul_pow_irreducible_factorization R → unique_factorization_monoid R`. If I rename it `unique_factorization_monoid` then the next lemma, `lemma of_ufd_of_unique_irreducible [unique_factorization_monoid R] ...` stops compiling because Lean thinks that `unique_factorization_monoid` refers to what we've just proved rather than the typeclass. I've renamed it `to_unique_factorization_monoid` to avoid this. Note that this makes this PR no longer 100% documentation, there is now alpha-equivalence involved.